scripts/ci/create-release.sh

```diff
@@ -37,30 +66,14 @@ changelog=$(scripts/ci/extract-changelog.py CHANGELOG.md $CHANGELOG_VERSION)
 make static WITH_EBPF=true
 ${dir}/../../contrib/packaging/rpm/generate-skydive-bootstrap.sh -s -r ${TAG}
 
-github-release release ${FLAGS} --user skydive-project --repo skydive --tag ${TAG} --description "$changelog"
-retcode=$?
-if [ $retcode != 0 ]; then
-    git push --delete origin $TAG || true
-    exit $retcode
-fi
-
-github-release upload --user skydive-project --repo skydive --tag ${TAG} --name skydive --file $GOPATH/bin/skydive
-retcode=$?
-if [ $retcode != 0 ]; then
-    git push --delete origin $TAG || true
-    github-release delete --user skydive-project --repo skydive --tag ${TAG} || true
-    exit $retcode
-fi
+uber=$GOPATH/src/github.com/skydive-project/skydive/contrib/exporters/uber
 
-github-release upload --user skydive-project --repo skydive --tag ${TAG} --name skydive-${VERSION}.tar.gz --file rpmbuild/SOURCES/skydive-${VERSION}.tar.gz
-retcode=$?
-if [ $retcode != 0 ]; then
-    git push --delete origin $TAG || true
-    github-release delete --user skydive-project --repo skydive --tag ${TAG} || true
-    exit $retcode
-fi
+release "$changelog"
+upload skydive $GOPATH/bin/skydive
+upload skydive-${VERSION}.tar.gz rpmbuild/SOURCES/skydive-${VERSION}.tar.gz
+upload skydive-exporters $uber/uber
+upload skydive-exporters-conf $uber/uber.yml.default
```

Comment:
change uber to 'exporter' [we are not in the taxi industry :-) ]

Reply:
fixed label (left source)